packages/hooks/src/useCheck10kLimit.ts

```diff
@@ -0,0 +1,21 @@
+import { useFetch } from '@deriv/api';
+import { useStore } from '@deriv/stores';
+import { getMinWithdrawal } from '../../shared/src/utils/currency';
```

Comment:
@aum-deriv Please don't import files cross packages 🙏🏻

Reply:
I believe we can even get rid of `withdrawal-locked` entirely once (this [PR](https://github.com/binary-com/deriv-app/pull/8867) is merged to master) and include the `useWithdrawalLocked` logic in `CashierLockedChecker` 🤔